### PR TITLE
Add Huey to comparison table

### PR DESCRIPTION
**Pull request by bersace**
Hi @Bogdanp and @coleifer,

I find the comparision table between Python task queues implementation very useful, thus I suggest to add huey in it. Huey is quite popular and has some nice features.

However, I am not sure of huey property compared to dramatiq for each point. So can you please review and explain if i'm wrong ?

I think it's good for both projects to compare. Also, the comparison table may be more fair and useful if more missing features from dramatiq were shown (aside Python2). For example, dramatiq does not have cron-like periodic scheduling while both Celery and Huey have.

Thanks for your help !

cf. #122

### CODE REVIEW

**Comment by Bogdanp** [comment]
> I think it's good for both projects to compare. Also, the comparison table may be more fair and useful if more missing features from dramatiq were shown (aside Python2). For example, dramatiq does not have cron-like periodic scheduling while both Celery and Huey have.

Feel free to add whatever feature you can think about to the table.

**Comment by coleifer** [comment]
If you want to add another one or two, huey supports

* task pipelines and chains (also supported by celery)
* supports multiprocess, multi-thread or greenlet execution models

Of course if you feel those aren't relevant to this comparison, its just a suggestion.

File: docs/source/motivation.rst [hunk 1]
```diff
@@ -68,6 +71,9 @@ of the main differences between Dramatiq, Celery and RQ:
 | In-memory broker support     | Yes      | No          | Yes          | No           |
 |                              |          |             |              |              |
 +------------------------------+----------+-------------+--------------+--------------+
+| Greenlet execution model     | No       | Yes         | Yes          | No           |
```
Comment (Bogdanp) [inline docs/source/motivation.rst]: Greenlets are supported in `Dramatiq` via gevent.

**Comment by bersace** [comment]
Updated.

@Bogdanp I though this is an opportunity to somewhat group criteria. I grouped reliability criteria together (retries, ack, locks)  and scheduling feature (delay, crontab, etc.).

Would you mind to review this ?

File: docs/source/motivation.rst [hunk 2]
```diff
@@ -35,28 +35,31 @@ of the main differences between Dramatiq, Celery and RQ:
...
-| Locks and rate limiting      | Yes      | No          | Yes          | No           |
+| Task prioritization          | Yes      | No [#prio]_ | No           | No [#prio]_  |
 |                              |          |             |              |              |
 +------------------------------+----------+-------------+--------------+--------------+
-| Result storage               | Yes      | Yes         | Yes          | Yes          |
+| Delayed tasks                | Yes      | Yes [#]_    | Yes          | No           |
 |                              |          |             |              |              |
 +------------------------------+----------+-------------+--------------+--------------+
-| Simple implementation        | Yes      | No [#sim]_  | Yes          | Yes          |
+| Chaining / Pipelining        | No       | Yes         | Yes          | No           |
```
Comment (Bogdanp) [inline docs/source/motivation.rst]: Dramatiq also has support for chaining.

**Comment by bersace** [comment]
Thanks @Bogdanp, PR updated.

**Comment by coleifer** [comment]
Huey column seems good to me.

File: docs/source/motivation.rst [hunk 3]
```diff
@@ -29,45 +29,54 @@ I've used Celery_ professionally for years and my growing frustration
...
+| RabbitMQ support             | Yes      | Yes         | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Redis support                | Yes      | Yes         | Yes          | Yes          |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| In-memory broker support     | Yes      | No          | Yes          | No           |
+|                              |          |             |              |              |
++------------------------------+----------+-------------+--------------+--------------+
+| Greenlet execution model     | Yes       | Yes         | Yes          | No           |
```
Comment (rpkilby) [inline docs/source/motivation.rst]: Minor nitpick: There's an extra space in the Dramatiq column that makes the row formatting inconsistent with the other rows.

Comment (bersace) [reply docs/source/motivation.rst]: Thanks @rpkilby !

Comment (rpkilby) [reply docs/source/motivation.rst]: 🙇

**Comment by coleifer** [comment]
Just an FYI, Huey now supports [task prioritization](https://huey.readthedocs.io/en/latest/guide.html#task-priority).

**Pull request closed without merging**